### PR TITLE
contrib/swww: new package (0.8.1)

### PR DESCRIPTION
**Pull request by Froggo8311**

### CODE REVIEW

File: contrib/swww/template.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pkgname = "swww"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [ "cargo" ]
+makedepends = [ "rust" ]
```
Comment (wezm) [inline contrib/swww/template.py:6]: CI is waiting on approval but the [`black`](https://pkgs.chimera-linux.org/package/current/main/x86_64/python-black) job will fail these lines. You'll need to format the template with it.

**Comment by JamiKettunen** [comment]
Please squash the format commit, you can e.g. `git rebase -i origin/master` and mark it as `f` for `fixup` (instead of default `pick`) which will merge it to the commit above (`contrib/swww: add package`)

**Comment by q66** [comment]
as @JamiKettunen said, commits need squashing (ditto for every other PR)

**Comment by Froggo8311** [comment]
@q66 Why not do it through github?

**Comment by q66** [comment]
because 1) you're the one who wants it merged, so you should be the one to put in the effort to make sure things are in a correct format 2) the commits are expected to have correct commit messages 3) it may not always be merged through github 4) ...

**Comment by Froggo8311** [comment]
Squashed & rebased

**Comment by q66** [comment]
fix commit message

**Comment by Froggo8311** [comment]
fixed

**Pull request merged**